ddl/column.go

```diff
@@ -535,6 +535,17 @@ func isColumnWithIndex(colName string, indices []*model.IndexInfo) bool {
 	return false
 }
 
+func isColumnWithForeignKey(colName string, fkInfos []*model.FKInfo) *model.FKInfo {
+	for _, fkInfo := range fkInfos {
```

Comment:
Feel like we can change the name to `isForeignKeyWithColumn` or `isColumnInForeignKey`

Reply:
Return value is `FKInfo` but not bool, so the function name should begin with `get`.